linkerd/core/src/main/scala/io/buoyant/linkerd/SvcConfig.scala

```diff
@@ -18,6 +18,7 @@ import io.buoyant.router.RetryBudgetModule.{param => ev}
  * class as part of config deserialization.
  */
 trait SvcConfig {
+  type ProtocolResponseClassifier = ResponseClassifier
```

Comment:
So the types of some of the defs in this trait can be polymorphic – `H2SvcConfig` overrides this type with `h2.ResponseClassifier`